src-11ty/redirect-page.html

```diff
@@ -63,6 +89,14 @@ <h1 class="card-header">Link account to Home Assistant?</h1>
   <h1 class="card-header">Open page in your Home Assistant?</h1>
   <div class="card-content">
     You've been linked to the page that will {{ redirect.description }}.
+    {% if redirect.custom %}
```

Comment:
Don't put it inside the card-content. I think above the header is better? (but kill the white space between blue top border and the warning)

https://deploy-preview-256--my-home-assistant.netlify.app/redirect/hacs_repository/?owner=asd&repository=asd

### Current

![image](https://user-images.githubusercontent.com/1444314/178060855-4ef28a1e-5d05-40a3-aae1-66d2c4bd8886.png)

### Moved higher

![image](https://user-images.githubusercontent.com/1444314/178060826-ff0b5df4-b557-498e-a394-d5df6164e47b.png)


Also looks like there needs to be more space between icon and text